android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -107,8 +107,8 @@ public VideoStreamingParameters getStreamingParams(){
     }
 
     @SuppressWarnings("unused")
-    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, VideoStreamingFormat format) {
-        this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format);
+    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, double scale, VideoStreamingFormat format) {
```

Comment:
This is changing a public method which cannot be done in a minor update. Please deprecate the old method and then add a separate one with this new signature

Reply:
DONE

Reply:
well, this needs to go out as a minor change, so what we can do is keep the old method, and simply add a new one with your new parameter. that makes it a minor change.

Reply:
So what you did here is correct, this is a minor change, which is what we were aiming for ✅